app-text/cmark/cmark-0.31.1-r1.ebuild

```diff
@@ -12,7 +12,7 @@ HOMEPAGE="https://github.com/commonmark/cmark"
 SRC_URI="https://github.com/commonmark/cmark/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="BSD-2"
-SLOT="0/0.31.0"
```

Comment:
If it needs to be changed on every release as upstream don't guarantee stable ABI, please do 0/${PV} instead.